### PR TITLE
Add SHOW_DEV_TOOLS environment variable

### PR DESCRIPTION
**Pull request by chamerling**
This PR allows to open devtools from the command line when launching the application such as:

```sh
SHOW_DEV_TOOLS=true npm start
```

It also adds the command line support when launching app (https://www.electronjs.org/docs/api/command-line) with `--show-dev-tools`

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ const path = require('path');
 const URL = require('url');
 const config = require('./app/features/config');
 
+const showDevTools = Boolean(process.env.SHOW_DEV_TOOLS);
```
Comment (saghul) [inline main.js:24]: How about doing a command line argument too?

File: package-lock.json [hunk 2]
```diff
@@ -4,106 +4,100 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "7zip-bin": {
```
Comment (saghul) [inline package-lock.json:7]: What's with this entire diff? You haven't changed package.json even :-O

File: main.js (hunk 1 above)
Comment (saghul) [reply main.js:24]: That would also work on release builds for example.

Comment (chamerling) [reply main.js:24]: Good idea, adding it

File: package-lock.json (hunk 2 above)
Comment (chamerling) [reply package-lock.json:7]: Umh, looks like my `npm update` changed things... Removing this file from the PR

File: main.js (hunk 1 above)
Comment (chamerling) [reply main.js]: Add command line support

File: package-lock.json (hunk 2 above)
Comment (chamerling) [reply package-lock.json]: Removed

File: main.js [hunk 3]
```diff
@@ -21,6 +21,8 @@ const path = require('path');
 const URL = require('url');
 const config = require('./app/features/config');
 
+const showDevTools = Boolean(process.env.SHOW_DEV_TOOLS) || app.commandLine.hasSwitch('show-dev-tools');
```
Comment (saghul) [inline main.js:24]: Wait, this is not going to work. Doesn't `app.commandLine`refer to the Chromium command line args?

Comment (chamerling) [reply main.js]: @saghul true, sorry for that. I updated the PR

**Comment by saghul** [comment]
Cheers!

**Pull request merged**